script/Arcueid/Day 10/10_00_ARC10_3B.txt

```diff
@@ -361,7 +361,7 @@ The best place to take her has to be―――
 [sha:c3eb2e37de11bba113e34a0dfd4303ab963c7fa2]{
 -- Page 120, Offset 42672. Choice.
 -- 定番だけど映画館はどうだろう。
- The movie theater, even if it is a little cliche.
+ The movie theater, even if it's a little cliche.
```

Comment:
I actually think we should keep/emphasize the `is` here. Keep separate and maybe even make it `%{i}is%{/i}`